src/ponyc/main.c

```diff
@@ -120,6 +122,8 @@ static void usage()
     "  --runtimebc     Compile with the LLVM bitcode file for the runtime.\n"
     "  --pic           Compile using position independent code.\n"
     "  --docs, -g      Generate code documentation.\n"
+    "  --docs-private  Generate code documentation for private types\n"
```

Comment:
nothing, at the moment. 
The compiler just does its passes (like it would do without).
I will ad a stance about it to the help string and add a warning output that it will be ignored.

Other possibility is to model this switch differently, having ``--docs-public`` generating docs for public types only and ``--docs`` generating docs for both, just like ``--docs`` did before.

Now that i think about it, adding ``--docs-public`` looks about right to me. Give me a day to implement the different behavior (and the formatting). :)